.github/workflows/wiki_label.yml

```diff
@@ -0,0 +1,51 @@
+name: 'PR Wiki Label'
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  wiki-label:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && github.event.comment.body == '!wiki_label'
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
```

Comment:
Do we need to checkout the repo for a comment action?

Reply:
Not anymore.
I will take it out on the next commit.

Reply:
New changes pushed. I will test locally again to make sure it still functions properly.